stream_chat/tests/test_client.py

```diff
@@ -10,6 +10,17 @@ class TestClient(object):
     def test_mute_user(self, client, random_users):
         response = client.mute_user(random_users[0]["id"], random_users[1]["id"])
         assert "mute" in response
+        assert "expires" not in response["mute"]
+        assert response["mute"]["target"]["id"] == random_users[0]["id"]
+        assert response["mute"]["user"]["id"] == random_users[1]["id"]
+        client.unmute_user(random_users[0]["id"], random_users[1]["id"])
+
+    def test_mute_user_with_timeout(self, client, random_users):
+        response = client.mute_user(
+            random_users[0]["id"], random_users[1]["id"], timeout=10
+        )
+        assert "mute" in response
+        assert "expires" in response["mute"]
```

Comment:
how about checking after timeout has passed ?

Reply:
The thing is that the timeout is in minutes, a 1 minute wait seemed a bit long to me for a test. Since the client-side implementation of the expiration is up to the dev, I thought I'd test at least that the backend returns an expiration timestamp as expected... open to suggestions though!

Reply:
sounds reasonable, good as it is